### PR TITLE
feat/added link preview component

### PR DESCRIPTION
**Pull request by DavidKorystov**
Example:
```
<LinkPreview url={"https://getbootstrap.com/docs/5.0/components/card/"} />
```
image

### CODE REVIEW

**Review by VladislavPokhyl — APPROVED** [review]

**Review by Mykyta-Pysarev — APPROVED** [review]
Looks good to me

File: src/components/LinkPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+interface PropsUrl {
+  url: string;
+}
+
+const Wrapper = styled.div`
+  width: 300px;
```
Comment (alexandrtovmach) [inline src/components/LinkPreview.tsx]: why magic 300px?

Comment (alexandrtovmach) [reply src/components/LinkPreview.tsx]: this component will be used inside another components, so it shouldn't shouldn't have limited width

File: src/components/LinkPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  const [responseObject, setResponseobject] = useState({
+    description: "",
+    image: "",
+    title: "",
+    url: "",
+  });
+
+  useEffect(() => {
+    fetch("https://api.linkpreview.net", {
```
Comment (alexandrtovmach) [inline src/components/LinkPreview.tsx]: move this logic to separate service "services/linkPreview.ts"

File: src/components/LinkPreview.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  return (
+    <Wrapper className="card">
+      <img src={responseObject.image} className="card-img-top" alt="preview" />
+
+      <div className="card-body ">
```
Comment (alexandrtovmach) [inline src/components/LinkPreview.tsx]: use react-bootstrap analouge conmponents instead of pure classes

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
few changes please

File: src/components/LinkPreview.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+  useEffect(() => {
+    fetch("https://api.linkpreview.net", {
+      method: "POST",
+      mode: "cors",
+      body: JSON.stringify(dataObject),
+    })
+      .then((res) => {
+        if (res.status != 200) {
+          // console.log(res.status);
```
Comment (ArtemLigerko) [inline src/components/LinkPreview.tsx]: please, remove comment

**Review by ArtemLigerko — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**